content/go/concepts/loops/loops.md

```diff
@@ -0,0 +1,138 @@
+---
+Title: 'Loops'
+Description: 'There are several different types of loops in Go. All are written using the `for` command using different syntax.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Control Flow'
+  - 'Iterators'
+  - 'Loops'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+There are several different types of loops in Go. All are written using the `for` command using different syntax.
+
+## Three Component Loop
+
+The three component loop syntax is the same as ones used in C or Java. This is what is most often called a `for` loop in other languages.
+
+### Syntax
+
+```pseudo
+for init; condition; post {
+  statements
+}
+```
+
+Here `init` is a statement initalizing an index variable. `condition` is a boolean expression checking the index variable before every loop, telling the loop to execute as long as it's true. And `post` is a statement executed after every loop to increment/decrement the index variable.
+
+### Example
```

Comment:
I think each example would benefit from having a brief explanation of what each does and the output for each.

Reply:
Done